### PR TITLE
Always suppress invoices

### PR DESCRIPTION
**Pull request by nickpell**
help avoid issues like http://manage.dimagi.com/default.asp?169320

filter for is_hidden_to_ops=False unless explicitly not set

@benrudolph @proteusvacuum @amsagoff cc: @NoahCarnahan 

wait on tests

### CODE REVIEW

File: corehq/apps/accounting/tests/test_invoicing.py [hunk 1]
```diff
@@ -460,9 +461,9 @@ def test_hide_invoices(self):
         # Basic hide invoices
         call_command('hide_invoices_by_id', *[i.pk for i in invoices])
         for i in invoices:
-            self.assertTrue(Invoice.objects.get(pk=i.pk).is_hidden_to_ops)
+            self.assertTrue(super(InvoiceBaseManager, Invoice.objects).get_queryset().get(pk=i.pk).is_hidden_to_ops)
```
Comment (dimagimon) [inline corehq/apps/accounting/tests/test_invoicing.py]: E501 line too long (116 > 115 characters)

File: corehq/apps/accounting/tests/test_invoicing.py [hunk 2]
```diff
@@ -460,9 +461,9 @@ def test_hide_invoices(self):
...
 
         # Basic unhide invoices
         call_command('hide_invoices_by_id', *[i.pk for i in invoices], unhide=True)
         for i in invoices:
-            self.assertFalse(Invoice.objects.get(pk=i.pk).is_hidden_to_ops)
+            self.assertFalse(super(InvoiceBaseManager, Invoice.objects).get_queryset().get(pk=i.pk).is_hidden_to_ops)
```
Comment (dimagimon) [inline corehq/apps/accounting/tests/test_invoicing.py]: E501 line too long (117 > 115 characters)

File: corehq/apps/accounting/management/commands/hide_invoices_by_id.py [hunk 3]
```diff
@@ -24,7 +24,7 @@ def handle(self, *args, **options):
 
         for invoice_id in args:
             try:
-                invoice = Invoice.objects.get(pk=invoice_id)
+                invoice = super(InvoiceBaseManager, Invoice.objects).get_queryset().get(pk=invoice_id)
```
Comment (benrudolph) [inline corehq/apps/accounting/management/commands/hide_invoices_by_id.py]: wait so does this mean we have to use this syntax every time we want to get an invoice?

Comment (nickpell) [inline corehq/apps/accounting/management/commands/hide_invoices_by_id.py]: only when you want to override the default filter ```is_hidden_to_ops=False```.  These instances should be rare (for example, now it's only required in one management command and in a test - every other ```Invoice.objects``` call precedes normally)

Comment (benrudolph) [inline corehq/apps/accounting/management/commands/hide_invoices_by_id.py]: :+1:

**Pull request merged**